packages/dds/tree/package.json

```diff
@@ -46,7 +70,7 @@
 		"test:snapshots:regen": "npm run test:mocha -- --snapshot",
 		"test:stress": "cross-env FUZZ_TEST_COUNT=20 FUZZ_STRESS_RUN=true mocha --ignore \"dist/test/memory/**/*\" --recursive \"dist/test/**/*.spec.js\" -r @fluidframework/mocha-test-setup",
 		"ts2esm": "ts2esm ./tsconfig.json ./src/test/tsconfig.json",
-		"tsc": "tsc",
+		"tsc": "tsc --project ./tsconfig.cjs.json && copyfiles -f ../../../common/build/build-common/src/cjs/package.json ./dist",
```

Comment:
I think this is fine for now since fluid-build understands tsc and copyfiles and should decompose this into two dependent tasks (one for tsc and the other for copyfiles). However, longer-term I think the pattern we should use is to have explicit separate tasks for the copying and make those dependencies explicit in the fluid-build config as well.

Reply:
I did enjoy fluid-build being smart about this. I am not sure the long-term desired breakup. I guess `tsc` would kick off the two tasks b/c we want to keep the main build command simple and common. So, we have tsc that does tsc:transpile and tsc:setcjs later?